pkg/commands/delete.go

```diff
@@ -46,6 +52,15 @@ func passthru(command string) runCmd {
 	}
 }
 
+// check if kubectl is installed
+func isKubectlAvailable() bool {
+	cmd := exec.Command("kubectl")
```

Comment:
You could use [`exec.LookPath`](https://golang.org/pkg/os/exec/#LookPath) for this, and we probably want to do this for `apply`, `create`, and `run` as well.

Reply:
@jonjohnsonjr Awesome! I can definitely use `LookPath` instead. Thanks for the better suggestion. Would you recommend a good place to define `isKubectlAvailable()` so I can use it across all commands?

Reply:
Anywhere is fine! I'd just leave it here, probably.

Reply:
I put in `commands.go` for now, but more than willing to move it back to any of the commands that use it.